legal-api/report-templates/amalgamationApplication.html

```diff
@@ -1,13 +1,241 @@
+    <div class="no-page-break">
+      <div class="section-header mt-5 pt-4">AMALGAMATION APPLICATION</div>
+    </div>
+
+    <!-- Name Request [[amalgamation/nameRequest.html]]-->
+    <div class="no-page-break">
+      <table class="section-data nr-name-table mt-4" role="presentation">
+          <tr>
+              <td class="bold col-33">Name Reservation Number:</td>
+              <td>
+                  {% if not amalgamationApplication.nameRequest.nrNumber %}
+                  N/A
+                  {% else %}
+                  {{amalgamationApplication.nameRequest.nrNumber}}
+                  {% endif %}
+              </td>
+          </tr>
+          <tr>
+              <td class="bold pt-2">Name Reserved:</td>
+              <td class="pt-2">
+                  {% if not amalgamationApplication.nameRequest.nrNumber %}
+                  The company is to be incorporated with a name created by adding "B.C. LTD." after the incorporation number.
+                  {% else %}
+                  {{amalgamationApplication.nameRequest.legalName}}
+                  {% endif %}
+              </td>
+          </tr>
+      </table>
+    </div>
+
+    <!-- Effective Date [[amalgamation/effectiveDate.html]]-->
+    <div class="no-page-break">
+      <div class="separator mt-4"></div>
+      <div class="section-title mt-4">Amalgamation Effective Date</div>
+      <div class="section-data mt-4">
+          {% if header.isFutureEffective %}
+              {{header.effective_date_time}}
+          {% else %}
+              The incorporation is to take effect at the time that this application is filed with the Registrar.
+          {% endif %}
+      </div>
+    </div>
+
+    <!-- Approval Type [[amalgamation/approvalType.html]]-->
+    <div class="no-page-break">
+      <div class="separator mt-4"></div>
+      <div class="section-title mt-4">
+        <span>Approval Type</span>
+      </div>
+      {% if amalgamationApplication.courtApproval is none %}
+        <div class="section-data mt-3">
+          N/A
+        </div>
+      {% else %}
+        {% if amalgamationApplication.courtApproval == 'courtOrder' %}
+          <div class="section-data mt-3">
+            The restoration was approved by a court order.
+          </div>
+          <div class="section-data mt-2">
+            Court order number: <span class="bold">{{amalgamationApplication.courtApproval.fileNumber}}</span>
+          </div>
+        {% elif amalgamationApplication.courtApproval == 'registrar' %}
+          <div class="section-data mt-3">
+            The restoration was approved by Registrar.
+          </div>
+          <div class="section-data mt-2">
+            BC Gazette published date: {{amalgamationApplication.courtApproval.noticeDate}}
+          </div>
+          <div class="section-data mt-2">
+            Notice of Application mailed date: {{amalgamationApplication.courtApproval.applicationDate}}
+          </div>
+        {% endif %}
```

Comment:
We don't have these details. I don't think we had a requirement (yet) to restore. right @argush3?

Reply:
yeah I think some of these elements in the amalgamationApplication might have been copied from elsewhere?  

I don't see these items in the design or maybe I'm referencing the incorrect [design link](https://preview.uxpin.com/7f4cfe5fe4b0ba15a83d518177efe82a08e99167#/pages/160577435/simulate/sitemap).

Reply:
Copied!

Reply:
oh i see you have the same bit down in the `approvalType.html`.  

Not sure what the plan is here.  are you going to update amalgamationApplication.html to use a bunch of the template html snippets under `template-parts/amalgamation`?

Reply:
Exactly, now I know how so I will have that up on my next push, should look a lot cleaner.